libs/api_common/windows_helpers.hpp

```diff
@@ -18,10 +21,11 @@ typedef struct empty_reply
 
 static empty_reply_t _empty_reply;
 
-template <typename request_t, typename reply_t = empty_reply_t>
-uint32_t
+template <typename request_t, typename reply_t = empty_reply>
+static uint32_t
 invoke_ioctl(ebpf_handle_t handle, request_t& request, reply_t& reply = _empty_reply)
```

Comment:
```suggestion
_invoke_ioctl(ebpf_handle_t handle, request_t& request, reply_t& reply = _empty_reply)
```
Prefix static functions with `_`

Reply:
This was actually a bad merge. invoke_ioctl is now moved to device_helper.cpp and its not static.